### PR TITLE
Docs: Update readme

### PR DESCRIPTION
**Pull request by MichaelCurrin**
- Fixed a typo
- Moved badges below title and center align
- Rename config to Field in tables and `ex` to `e.g.`
- Added link to screenshot and made it a local link not a raw link
- Updated italics, bold, bullet points and check boxes for consistency with markdown best practice
- Used H2 headings in place of italics

### CODE REVIEW

**Review by tcbutler320 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,120 +1,115 @@
...
-    <a href="https://dumbarton.netlify.app/">View Demo</a>
-    ·
-    <a href="https://github.com/tcbutler320/Jekyll-Theme-Dumbarton/issues">Report Bug</a>
-    ·
-    <a href="https://github.com/tcbutler320/Jekyll-Theme-Dumbarton/issues">Request Feature</a>
-  </p>
-</p>
+![Bootstrap](https://img.shields.io/badge/Bootstrap-v4.5.3-blue) 
 
-![](https://raw.githubusercontent.com/tcbutler320/Jekyll-Theme-Dumbarton/dev/screenshot.png)
```
Comment (tcbutler320) [inline README.md:25]: I typically would not use raw images on the readme, however this was done to satisfy [Jekyll-Themes.com](https://jekyll-themes.com/submit/). For whatever reason, they require links on the readme to be raw links.  

Screen Shot 2020-11-24 at 8 56 20 PM

**Review by tcbutler320 — CHANGES_REQUESTED** [review]
Thanks so much for the input @MichaelCurrin! I'm ready to approve all your changes except for the change in image link format on the readme. The raw link was used to satisfy [Jekyll-Themes.com ](https://jekyll-themes.com/submit/). For whatever reason they use raw links and I will eventually make a PR to get Dumbarton on their site. 

If you can just adjust this and resubmit your PR I'll merge your changes.

File: README.md (hunk 1 above)
Comment (MichaelCurrin) [reply README.md:25]: Aha. They must be pulling the README as a preview on their site, so a relative link on GitHub won't work without them doing extra to it.

No problem, I'll adjust

**Review by tcbutler320 — APPROVED** [review]

**Comment by tcbutler320** [comment]
Merged, thanks for the updates @MichaelCurrin !

**Pull request merged**